react/features/filmstrip/components/native/RaisedHandIndicator.js

```diff
@@ -0,0 +1,42 @@
+/* @flow */
+
+import React from 'react';
+import { View } from 'react-native';
+
+import { Icon } from '../../../base/font-icons';
+import { connect } from '../../../base/redux';
+
+import AbstractRaisedHandIndicator, {
+    type Props,
+    _mapStateToProps
+} from '../AbstractRaisedHandIndicator';
+
+import styles from './styles';
+
+/**
+ * Thumbnail badge showing that the participant would like to speak.
+ *
+ * @extends Component
+ */
+class RaisedHandIndicator extends AbstractRaisedHandIndicator<Props> {
```

Comment:
Side comment: how about some base class for all indicators? They must all be the same: have a way to check if they should be shown or not, then render an icon name which could be a class property. Just a thought.

Reply:
Good idea, done, thanks